### DISCUSSION TITLE
How to plot grid line for a GridMapDisplayBasemap?

### DISCUSSION THREAD

**Discussion by yhp92** [ARM-DOE/pyart#1436] [Q&A] [answered]
Hi, so for RadarDisplay class there is plot_grid_lines function. I was wondering what is the way to plot the grid lines for a GridMapDisplayBasemap? 

Thank you!

**Comment by ghost** [comment]
Hello Hanpu Yao you my use following method
GridMapDisplayBasemap.plot_crosshairs(*lon=None*, *lat=None*,
*line_style='r--'*, *linewidth=2*, *ax=None*)

Regards ,
Yogesh Kolte.

**Reply by yhp92** [reply]
Thank you for your answer, Yogesh! What does all the * mean?

**Comment by yhp92** [comment, marked as answer]
Seems hardcoding the lat_lines and lon_lines works
`GridMapDisplayBasemap.plot_basemap(resolution='l',lat_lines=lat_lines, lon_lines=lon_lines)`
